package-support-policy.md

```diff
@@ -0,0 +1,90 @@
+# Support Policy for .NET Packages
+
+NuGet packages owned by the .NET team are supported as long as they target at least one framework that is still actively supported.
+
+The minimum supported .NET versions are:
+
+- .NET Framework 4.6.2
+- .NET Standard 2.0
+- .NET 6.0
+
+The support policy of our frameworks can be found here: https://dotnet.microsoft.com/en-us/platform/support/policy
+
+
+## Package and project compatibility
+
+A package targeting any of these above mentioned supported frameworks is expected to be compatible with projects targeting any newer versions. For example:
+
+- If you have a project that targets .NET 8.0, and it consumes a package that targets either .NET Standard 2.0 or .NET 6.0, you should expect the package to be compatible with your project because .NET Standard 2.0 supports .NET 8.0 and .NET 8.0 is greater than .NET 6.0.
```

Comment:
Maybe it could be made in table form intead of text?
like Package TFM, Project TFM, Comment

probably it will be easiear to understand

Reply:
Interesting suggestion. Although based on the rest of the feedback, it seems the version compatibility can be understood by looking at the .NET Standard document. I can just link to that and keep it simpler: no list and no table needed.